specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/storage.json

```diff
@@ -0,0 +1,1921 @@
+        "tags": [
+          "StorageAccounts"
+        ],
+        "operationId": "StorageAccounts_ListKeys",
+        "description": "Lists the access keys for the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountListKeys": {
+            "$ref": "./examples/StorageAccountListKeys.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- list of keys retrieved and returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/StorageAccountListKeysResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/regenerateKey": {
+      "post": {
+        "tags": [
+          "StorageAccounts"
+        ],
+        "operationId": "StorageAccounts_RegenerateKey",
+        "description": "Regenerates one of the access keys for the specified storage account.",
+        "x-ms-examples": {
+          "StorageAccountRegenerateKey": {
+            "$ref": "./examples/StorageAccountRegenerateKey.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "name": "regenerateKey",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StorageAccountRegenerateKeyParameters"
+            },
+            "description": "Specifies name of the key which should be regenerated -- key1 or key2."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- specified key regenerated successfully.",
+            "schema": {
+              "$ref": "#/definitions/StorageAccountListKeysResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Storage/locations/{location}/usages": {
+      "get": {
+        "tags": [
+          "LocationUsage"
+        ],
+        "operationId": "Usages_ListByLocation",
```

Comment:
I feel we should keep “Usage” in 2018-07-01 and not “Usages”. It was like that for the 6 APIs between 2015-06-15 and 2018-02-01, and was changed in 2018-03-01-preview. I feel like we should keep the naming that all stable API were using

Reply:
@lmazuel 
The change is required by @anuchandy in review https://github.com/Azure/azure-rest-api-specs/pull/3014, since Using non-plural name for operation group is against SDK guideline . 
And finally we fix it in a new API version 2018-03-01-preview in https://github.com/Azure/azure-rest-api-specs/pull/3137, since this is a breaking change.
So I don't think we should keep using "Usages", instead of change it back.

Reply:
Fair enough :)